.github/workflows/gerrit-compose-required-rtdv3-merge.yaml

```diff
@@ -79,6 +79,8 @@ jobs:
         id: setup-python
         with:
           python-version: "3.8"
+      - name: Install graphviz
+        uses: tlylt/install-graphviz/action.yml@v1.0.0
```

Comment:
There's no reason to specify action.yml here. In fact, it may not work as intended...

Reply:
updated